Mathlib/RingTheory/Ideal/Maps.lean

```diff
@@ -1067,6 +1067,12 @@ lemma comap_ker {C : Type*} [Semiring C] [Algebra R C] (f : B →ₐ[R] C) (g :
     (RingHom.ker f).comap g = RingHom.ker (f.comp g) :=
   RingHom.comap_ker f.toRingHom g.toRingHom
 
+/-- Synonym for `RingHom.ker_coe_equiv`, but given an algebra equivalence. -/
+@[simp] theorem ker_coe_equiv {R A B : Type*} [CommSemiring R] [Ring A] [Ring B]
```

Comment:
Should this go next to coe_ker above? (And shouldn't that be called ker_coe? Not proposing to change that here.)

Reply:
I agree with both naming points, but let's do that for all the other variants simultaneously.

As for location, I think it's better to put them next to the other `ker_coe` variants (even if they are in a different namespace).

Reply:
Both fixed.